fedimint-core/src/admin_client.rs

```diff
@@ -216,6 +216,15 @@ pub enum ServerStatus {
     ConsensusRunning,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct FullStatus {
+    pub our_epoch: u64,
+    pub last_epoch_by_peer: HashMap<PeerId, u64>,
+    pub peers_offline: u64,
+    pub peers_not_synchronized: u64,
```

Comment:
`peers_behind`, `peers_stale`, `peers_out_of_sync`?

Reply:
I would lean into `peers_out_of_sync`. The metric is how many peers there are with `status != FullPeerStatus::Synchronized`

Reply:
The thing is that `_not_` is kind meh, because generally one wants to avoid negations in names. *But* by using a negation we make it easier for simple monitoring tools t match that the response contains `"peers_out_of_sync": 0` literally. So it's preferable that all the stats are `0 == good`.

BTW. Might be worthwhile to put this context in a comment somewhere.